docs/content/concepts/authentication/index.md

```diff
@@ -0,0 +1,73 @@
+---
+description: >
+  How to authenticate requests to kcp.
+---
+
+# Authentication
+
+KCP implements the same authentication mechanisms as Kubernetes, allowing the use of Kubernetes authentication strategies. The KCP server can also be configured to generate a local admin.kubeconfig file and a token hash file, enabling access to KCP as a shard admin. This authentication mechanism is then added to any existing Kubernetes authentication strategies from generic control plane settings.
+
+## KCP server admin authentication
+
+Admin Authenticator sets up user roles and groups and generates authentication tokens. The authentication process relies on Kubernetes authenticated group authenticator.
+
+### Users and Groups
+
+| **User Name**   | **Role**                                                                                                                           | **Groups**                           |
+|-----------------|------------------------------------------------------------------------------------------------------------------------------------|--------------------------------------|
+| **shard-admin** | Member of the privileged system group. This user bypasses most KCP authorization checks.                                           | system:masters, system:authenticated |
+| **kcp-admin**   | Member of the system:kcp:workspace:admin and system:kcp:workspace:access groups. This user is subject to KCP authorization checks. | system:kcp:workspace:admin           |
+| **user**        | Regular non-admin user who is not a part of any predefined groups.                                                                 | None                                 |
```

Comment:
We should make it clear that this `admin.kubeconfig` is primarily generated when you run kcp locally in development mode. It's e.g. disabled in the Helm chart. So I think this section has to be more clear on "when does this kubeconfig get generated" and should be moved to the bottom of the page since most kcp admins will never use it.

Reply:
You are right, I will cover this in describing batteriesIncluded setting that is required to set this up